### PR TITLE
openxray: update package description

### PR DESCRIPTION
**Pull request by Xottab-DUTY**
###### Motivation for this change
To correspond the current project description.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by drewrisinger — APPROVED** [review]
* [x] Diff LGTM. Matches GitHub description
* [x] Commits LGTM
* [ ] Shouldn't rebuild, didn't test.

File: pkgs/games/openxray/default.nix [hunk 1]
```diff
@@ -66,7 +66,7 @@ in stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    description = "X-Ray Engine 1.6 expansion. Original version was used in S.T.A.L.K.E.R.: Call of Pripyat";
+    description = "Improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World.";
```
Comment (SuperSandro2000) [inline pkgs/games/openxray/default.nix:69]:
```suggestion
    description = "Improved version of the X-Ray Engine, the game engine used in the world-famous S.T.A.L.K.E.R. game series by GSC Game World";
```

Comment (Xottab-DUTY) [reply pkgs/games/openxray/default.nix:69]: Disagree on the dot. To me, a missing dot is grammatically rude. This is the end of the description and nothing will follow after it. So, it should end with the dot. It is not a funny chat, it is package description!
Also missing dot does not correspond with the project description.

But I can agree with you, if the's a good reason for removing the dot, eg. it looks ugly when you read the description in the terminal or in GUI, or, well, if all the packages don't have the dot in the end of the description...

Comment (SuperSandro2000) [reply pkgs/games/openxray/default.nix:69]: > Disagree on the dot.

Thats the rule. Short descriptions are no sentences and contain no final dot. No arguing.

Comment (Xottab-DUTY) [reply pkgs/games/openxray/default.nix]: Oops, sorry, I've missed that part of the CONTRIBUTING.md 😄

**Comment by Xottab-DUTY** [comment]
Everything is fixed.

**Pull request merged**